src/Symfony/Component/HttpFoundation/Tests/CookieTest.php

```diff
@@ -151,4 +151,13 @@ public function testToString()
         $cookie = new Cookie('foo', 'bar', 0, '/', '');
         $this->assertEquals('foo=bar; path=/; httponly', $cookie->__toString());
     }
+
+    public function testRawCookie()
```

Comment:
I don't know the way to capture headers in a test. Do you have an idea how to do it?